frontend/src/components/Forum/Post.jsx

```
@@ -64,64 +67,107 @@ const Post = ({ post, onDelete }) => {
+    {/* Post Title */}
+    <h2 className="text-xl font-semibold text-gray-900 mb-3">{updatedPost.title}</h2>
+
+    {/* Post Content */}
+    <div className="mb-4 text-gray-700 leading-relaxed">
+  {isExpanded ? updatedPost.content : `${updatedPost.content.slice(0, 250)}...`}
+  {updatedPost.content.length > 250 && (
+    <button 
+      onClick={toggleExpand} 
+      className="text-blue-600 font-medium text-sm ml-2 hover:underline hover:text-blue-800 transition"
+    >
+      {isExpanded ? "See Less" : "See More"}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)